sdc/rabbit/consumers.py

```diff
@@ -383,40 +383,74 @@ def stop(self):
         self.stop_consuming()
         logger.info('Stopped')
 
+    def _delay_before_reconnect(self):
+        """Sleeps for a number of seconds equal to the `count` attribute on this object
+        """
+        logger.info("Sleeping before reconnect", no_of_seconds=self._count)
+        time.sleep(self._count)
+        self._count += 1
+        logger.info("Finished sleeping")
```

Comment:
Do you need to say this? Or should the caller of this function just log that it's doing the thing it was doing again when it returns?

Reply:
Yeah that's fair.  Removed it